### PR TITLE
Invoice Statement Fixes and others

### PR DESCRIPTION
**Pull request by sjpadgett**
Bug fix to prevent appending file size to statement PDF. Reformat of statement footer to center and justify better.

@bradymiller Wouldn't hurt to leave open a couple days as may be others.

### CODE REVIEW

**Comment by sjpadgett** [comment]
@bradymiller I fixed the datetimepicker to default to not scroll months but having trouble with the alternative date picker. How do the vars get set in jquery-datetimepicker-2-5-4-alternate.js.php. i.e need:
var datepicker_scrollInput = false;
var datepicker_scrollMonth = false;
in  jquery-datetimepicker-2-5-4.php i just set scrollInput:false in init array. How is alternative used?

**Comment by bradymiller** [comment]
hi @sjpadgett ,
Added the alternate js script to work with your portal code. It basically just creates javascript variables that are then used downstream in your portal code. For the variables you added, you basically then use them in the $('.jquery-date-time-picker').datetimepicker({ function call in the portal/patient/scripts/app/patientdata.js script
(just let me know if nee more clarification)
-brady

File: sites/default/statement.inc.php [hunk 1]
```diff
@@ -403,15 +403,15 @@ function create_HTML_statement($stmt)
     }
```
Comment (stephenwaite) [inline sites/default/statement.inc.php]: how about at the end of line 394 
`xlt('Chart'); ?></b> <?php echo text($stmt['pid'])`
so the office can easily find the appropriate chart to post the payment
thanks Jerry

Comment (sjpadgett) [reply sites/default/statement.inc.php]: Okay, but I added my little gravitas :)
@bradymiller Yes, should be xlt() but was going to use a colon and forgot...

Comment (bradymiller) [reply sites/default/statement.inc.php]: @sjpadgett , you are forgiven :)

**Comment by bradymiller** [comment]
hi @sjpadgett ,
Just let me know when this PR is ready to go in and I'll bring it in.
-brady

**Comment by sjpadgett** [comment]
Thanks @bradymiller , If not a problem leave open but, if you don't mind me making small PR's as I come across bugs then by all means, bring in to keep base up to date. I don't mind doing PR's if admin's don't mind.

**Comment by bradymiller** [comment]
hi @sjpadgett ,
Bring the PRs on. There can never be too many PRs :)
(Single bug/issue PR are actually easier for the admins; I'll bring in this PR)
-brady

**Pull request merged**